internal/cmd/crc32_table/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+const (
+	disclaimer = `// Code generated by astits. DO NOT EDIT`
+	packet     = `package astits`
+	filename   = `crc32_table.go`
+)
+
+func main() {
+	file, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_TRUNC|os.O_SYNC, 0644)
+	if err != nil {
+		panic(err)
```

Comment:
Could you use `log.Fatal(err)` instead ?

Reply:
Done